jemalloc-sys/README.md

```diff
@@ -150,6 +150,14 @@ hyphens `-` are replaced with underscores `_`(see
   virtual address size on those platforms where it knows how, and picks a
   default otherwise. This option may be useful when cross-compiling.
 
+By default this crate builds and links against a vendored version of `jemalloc`. To
+provide your own `jemalloc` version, set `JEMALLOC_OVERRIDE` to point to a built
+version of `jemalloc`. This can point to either a shared or static library. For
+instance: `JEMALLOC_OVERRIDE=/path/to/libjemalloc.a`. The library should be built
+with `jemalloc`'s `--with-jemalloc-prefix=_rjem_`. For static libraries it is also
```

Comment:
It's not accurate. Whether prefix is required depends on whether `unprefixed_malloc_on_supported_platforms` is enabled and whether the target is compatible with overwriting malloc.

Reply:
> It's not accurate.

Right, though the default seems to be to use the prefixed version so I thought this was simpler. Should I mention the prefix logic here? I'd need to dig into it a bit more.

Reply:
How about changing it like following?

`JEMALLOC_OVERRIDE` is for advanced usage only. Providing your own library means most features of jemalloc-sys will be ignored in build script, and you are responsible to compile the library to match what jemalloc-sys expects. Especially, handle API prefix as whatever feature `unprefixed_malloc_on_supported_platforms` says.